### ISSUE TITLE
Per workspace wallpaper

### ISSUE THREAD

**Issue by kkua22** [WayfireWM/wayfire#800]

**Comment by ammen99** [comment]
Yes this is possible, however you need a custom plugin. Maybe @soreau will be willing to extend his already existing plugin to support this usecase.
